api/src/Metadata/Resource/Factory/UriTemplateFactory.php

```diff
@@ -2,27 +2,37 @@
 
 namespace App\Metadata\Resource\Factory;
 
-use ApiPlatform\Core\Action\NotFoundAction;
-use ApiPlatform\Core\Api\IriConverterInterface;
-use ApiPlatform\Core\DataProvider\PaginationOptions;
-use ApiPlatform\Core\Exception\ResourceClassNotFoundException;
-use ApiPlatform\Core\Metadata\Resource\Factory\ResourceMetadataFactoryInterface;
-use ApiPlatform\Core\Metadata\Resource\Factory\ResourceNameCollectionFactoryInterface;
-use ApiPlatform\Core\Metadata\Resource\ResourceMetadata;
+use ApiPlatform\Api\IriConverterInterface;
+use ApiPlatform\Api\UrlGeneratorInterface;
+use ApiPlatform\Exception\ResourceClassNotFoundException;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\Metadata\HttpOperation;
+use ApiPlatform\Metadata\Resource\Factory\ResourceMetadataCollectionFactoryInterface;
+use ApiPlatform\Metadata\Resource\Factory\ResourceNameCollectionFactoryInterface;
+use ApiPlatform\Metadata\Resource\ResourceMetadataCollection;
+use ApiPlatform\State\Pagination\PaginationOptions;
+use App\Metadata\Resource\OperationHelper;
 use Psr\Container\ContainerInterface;
 
+/**
+ * Creates templates URIS for resources.
```

Comment:
```suggestion
 * Creates templated URIs for resources.
```

But probably could be worded more clearly:
```suggestion
 * Given an entity class, creates an URI template (link with placeholders) for accessing this type of entity in the API.
```

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/3176/commits/dfb0b16cef2304a2b95128b69eebea07f3699e82